tests/pipelines/main.nf.test

```diff
@@ -1330,4 +1333,95 @@ nextflow_pipeline {
 
         }
     }
+    test("Test prefix parameter: output_prefix"){
+        tag "output_prefix"
+        when {
+            params {
+                input = "$baseDir/tests/data/samplesheets/samplesheet.csv"
+                outdir = "results"
+
+                metadata_partition_name = "outbreak"
+                metadata_1_header = "organism"
+                metadata_2_header = "subtype"
+                metadata_3_header = "country"
+                metadata_4_header = "serovar"
+                metadata_5_header = "age"
+                metadata_6_header = "date"
+                metadata_7_header = "source"
+                metadata_8_header = "special"
+
+                output_prefix     = "test"
+            }
+        }
+
+        then {
+
+            def date = LocalDate.now()
```

Comment:
For testing, I'm wondering if you should just check the date string against a regex rather than against the `LocalDate.now()`? Otherwise, it's possible (though rare) for these tests to fail if they are run right around midnight.